third-party/realdds/include/realdds/dds-stream-profile.h

```diff
@@ -69,12 +71,17 @@ struct dds_stream_format
 };
 
 
+class dds_stream_base;
+
+
 class dds_stream_profile
 {
     dds_stream_uid _uid;

```

Comment:
We did not finish our talk this morning. Do we need a stream_uid if we have pointer to the stream?

Reply:
No we do not.
But that's not what this PR does.
The stream/profile definitions will all change.